src/style/layout.css

```diff
@@ -296,7 +296,6 @@ body {
   position: fixed;
   color: var(--theme-foreground-muted);
   font: 400 14px var(--sans-serif);
-  z-index: 1;
```

Comment:
The TOC links are currently not clickable without the z-index: 1 because they are obscured (invisible) by the main column. We’ll need to fix this ASAP, perhaps by putting the observable-made-by within its own z-index container.

Reply:
Thanks @mbostock